### ISSUE TITLE
Prediction Accuracy on Nvidia Gesture dataset is very poor.

### ISSUE THREAD

**Issue by SumanDeedwaniya** [ahmetgunduz/Real-time-GesRec#43]
Hi Ahmet Gündüz,

I was using you pre-trained model (nv_resnext_101_Depth_32.pth) to test on nvidia gesture dataset. My accuracy for this dataset is very poor (not even 20%). Can you explain whether the model is correct one to test and if yes than why the prediction accuracy is so poor.
I have followed the steps mentioned by you in your github post.

**Comment by ahmetgunduz** [comment]
Hi, are you providing the model as `pretrain_path` or `resume_path`?

**Comment by terasakisatoshi** [comment]
- Hi, @ahmetgunduz Here is my output for nvGesture dataset with the following script.

```bash
python3 main.py \
	--root_path /work \
	--annotation_path annotation_nvGesture/nvall.json \
	--result_path results \
	--resume_path models/nv_resnext_101_Depth_32.pth \
	--dataset nv \
	--sample_duration 32 \
    --learning_rate 0.01 \
    --model resnext \
	--model_depth 101 \
	--resnet_shortcut B \
	--batch_size 64 \
	--n_classes 25 \
    --n_finetune_classes 25 \
	--n_threads 16 \
	--checkpoint 1 \
	--modality Depth \
	--train_crop random \
	--n_val_samples 1 \
	--test_subset test \
    --n_epochs 100 \
    --no_train \
    --no_val \
    --test \
```

```
[INFO]: NV Dataset - validation is loading...
dataset loading [0/964]
test
[1/16]	Time 8.064 (8.064)	Data 5.233 (5.233)	Acc 0.391 (0.391)
[2/16]	Time 0.576 (4.320)	Data 0.000 (2.616)	Acc 0.328 (0.359)
[3/16]	Time 0.598 (3.079)	Data 0.000 (1.744)	Acc 0.344 (0.354)
[4/16]	Time 0.576 (2.453)	Data 0.000 (1.308)	Acc 0.391 (0.363)
[5/16]	Time 0.576 (2.078)	Data 0.000 (1.047)	Acc 0.391 (0.369)
[6/16]	Time 0.576 (1.828)	Data 0.000 (0.872)	Acc 0.438 (0.380)
[7/16]	Time 0.576 (1.649)	Data 0.000 (0.748)	Acc 0.484 (0.395)
[8/16]	Time 0.576 (1.515)	Data 0.000 (0.654)	Acc 0.484 (0.406)
[9/16]	Time 0.576 (1.410)	Data 0.000 (0.581)	Acc 0.469 (0.413)
[10/16]	Time 0.577 (1.327)	Data 0.000 (0.523)	Acc 0.406 (0.412)
[11/16]	Time 0.577 (1.259)	Data 0.000 (0.476)	Acc 0.391 (0.411)
[12/16]	Time 0.577 (1.202)	Data 0.000 (0.436)	Acc 0.422 (0.411)
[13/16]	Time 0.577 (1.154)	Data 0.000 (0.403)	Acc 0.438 (0.413)
[14/16]	Time 0.577 (1.113)	Data 0.000 (0.374)	Acc 0.438 (0.415)
[15/16]	Time 0.576 (1.077)	Data 0.000 (0.349)	Acc 0.469 (0.419)
[16/16]	Time 0.075 (1.014)	Data 0.000 (0.327)	Acc 0.000 (0.417)
```

- Is this a correct result what we expect for nvGesture dataset ?

**Comment by ahmetgunduz** [comment]
No, It should be much higher. It has been a long time but As I remember you need to additionally add `pretrain_path` which is the same as `resume-path`.

**Comment by terasakisatoshi** [comment]
@ahmetgunduz Thank you for your comment!
I will try again 👍
